File: tfh-backend/src/main/java/com/example/tfhbackend/dto/MenuItemWithCountDTO.java
package com.example.tfhbackend.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Builder
@Data
public class MenuItemWithCountDTO {
    private MenuItemDTO item;
    private int count;
    private double price;
}

File: tfh-mobile/app/src/main/java/com/example/tfhmobile/dto/MenuItemWithCountDTO.kt
package com.example.tfhmobile.dto

class MenuItemWithCountDTO constructor(
    private val item: MenuItemDTO,
    private val count: Int,
    private val price: Double
) {
    fun getItem(): MenuItemDTO {
        return item;
    }

    fun getCount(): Int {
        return count
    }

    fun getPrice(): Double {
        return price
    }
}